integration_tests/run.sh

```diff
@@ -11,27 +11,43 @@ source ~/.bashrc
 conda env update -n jenkins_p2g_env -f build_environment.yml --prune
 conda activate jenkins_p2g_env
 
-time=`date +"%Y%m%d-%H%M%S"`
-tarball_name=polar2grid-swbundle-$time
+# Handle release vs test naming.
+end="`date +"%Y%m%d-%H%M%S"`"
+prefix="$(cut -d'-' -f1 <<<"$GIT_TAG_NAME")"
+version="$(cut -d'-' -f2 <<<"$GIT_TAG_NAME")"
+# If the tag is correct, make a version release.
+if [[ ("$prefix" = p2g || "$prefix" = g2g) && ! -z "$version" && "$prefix" != "$version" ]]; then
+    end="$version"
+fi
+if [[ "$prefix" = "g2g" ]]; then
+    prefix=geo
+else
+    prefix=polar
+fi
+tarball_name=${prefix}2grid-swbundle-$end
+
 ./create_conda_software_bundle.sh $WORKSPACE/$tarball_name
 export POLAR2GRID_HOME="$WORKSPACE/$tarball_name"
 cd "$WORKSPACE/integration_tests"
 behave --no-logcapture --no-color --no-capture -D datapath=/data/users/kkolman/integration_tests/polar2grid/integration_tests/p2g_test_data
 
 # Only ran by Jenkins if build was successful.
 # Save software bundle.
-rm -rf /tmp/polar2grid-*
-mkdir /tmp/polar2grid-$time
-cp -r "$WORKSPACE/$tarball_name" /tmp/polar2grid-$time
+rm -rf /tmp/${prefix}2grid-*
+mkdir /tmp/${prefix}2grid-$end
+cp -r "$WORKSPACE/$tarball_name" /tmp/${prefix}2grid-$end
 # Make docs.
-conda install -y sphinx
+conda update -c conda-forge -n jenkins_p2g_env sphinx
 pip install sphinx-argparse
 cd "$WORKSPACE"/doc
-make latexpdf
-cp -r "$WORKSPACE"/doc/build/latex /tmp/polar2grid-$time
+make latexpdf POLAR2GRID_DOC=$prefix
+cp -r "$WORKSPACE"/doc/build/latex /tmp/${prefix}2grid-$end
 # Clear out intermediate results and rebuild for HTML document.
 make clean
 # Needs to be second since Jenkins makes an html in workspace from the file generated by this command.
-make html
-cp -r "$WORKSPACE"/doc/build/html /tmp/polar2grid-$time
-chmod a+rx /tmp/polar2grid-$time
\ No newline at end of file
+make html POLAR2GRID_DOC=$prefix
+cp -r "$WORKSPACE"/doc/build/html /tmp/${prefix}2grid-$end
+chmod a+rx /tmp/${prefix}2grid-$end
+if [[ $end = "$version" ]]; then
+    cp -r /tmp/${prefix}2grid-$end /data/tmp/${prefix}2grid-$end
```

Comment:
If the build is versioned and tests succeeded could we copy the resulting `.tar.gz` file to `/data/dist`?